CHANGELOG.md

```diff
@@ -97,6 +100,8 @@ https://docs.docker.com/engine/deprecated/ where you can find the target removal
 + Add API support for templated secrets and configs [moby/moby#33702](https://github.com/moby/moby/pull/33702) and [moby/moby#36366](https://github.com/moby/moby/pull/36366)
 * Use rslave propagation for mounts from daemon root [moby/moby#36055](https://github.com/moby/moby/pull/36055)
 + Add /proc/keys to masked paths [moby/moby#36368](https://github.com/moby/moby/pull/36368)
+* Bump Runc to 1.0.0-rc5 [moby/moby#36449](https://github.com/moby/moby/pull/36449)
```

Comment:
Should probably mention the issues it addresses (devices getting wrong permissions, and exec issues on big-endian (s390x); but double check what the issues where 😅 )